Libraries/LibJS/Bytecode/IdentifierTable.h

```diff
@@ -34,3 +35,102 @@ class IdentifierTable {
 };
 
 }
+
+namespace AK {
```

Comment:
This can of course be implemented later, and I might even do it myself, but we should be able to use `Traits` (or a free function for `enum`) to generalize this behavior and avoid specializing `Optional` in multiple places.

Reply:
That's a neat idea. I'll leave it as a separate exercise :)